apps/src/weblab/CdoBramble.js

```diff
@@ -5,6 +5,33 @@ import {FatalErrorType} from './constants';
 import logToCloud from '../logToCloud';
 import {Buffer} from 'buffer';
 
+import testImageAccess from '../code-studio/url_test';
+import firehoseClient from '../lib/util/firehose';
+
+function tempLog(event, data = null) {
+  //console.log(event, data || "");
+
+  firehoseClient.putRecord(
+    {
+      study: 'weblab_loading_investigation_2022',
+      event: event,
+      data: data,
+      is_cros: /\bCrOS\b/.test(navigator.userAgent)
+    },
+    {includeUserId: true}
+  );
+}
+
+function testReach() {
+  testImageAccess(
+    'https://downloads.computinginthecore.org/favicon.ico' +
+      '?' +
+      Math.random(),
```

Comment:
Is the `Math.random()` here to prevent cached URLs? should use `Date.now()` in that case?

Reply:
Yes, though for better or worse, we seem to use `Math.random()` for most of our `testImageAccess` calls.

Reply:
Gotcha, yeah not a big deal if we have precedent of using it elsewhere.